src/gui/tray/UserModel.cpp

```diff
@@ -413,15 +413,23 @@ void User::slotAddError(const QString &folderAlias, const QString &message, Erro
     }
 }
 
+bool User::isValueableActivity(Folder *folder, const SyncFileItemPtr &item)
+{
+    // Check if we are adding it to the right account and if it is useful information (protocol errors)
+    const auto correctAccount = folder->accountState() == _account.data();
+    const auto conflictFromConflictedCopy = item->_status == SyncFileItem::Conflict && Utility::isConflictFile(item->_file);
+
+    return correctAccount && conflictFromConflictedCopy;
```

Comment:
@FlexW I can see a potential problem here. What would happen if the item, being added, is not with **_status == SyncFileItem::Conflict**? I guess, we should not affect other error activities, right? In that case, the following would work better:
`return correctAccount && (item->_status != SyncFileItem::Conflict || Utility::isConflictFile(item->_file));
`
and then, you also get rid of `conflictFromConflictedCopy`.

Reply:
@FlexW Optionally, you can put `item->_status != SyncFileItem::Conflict || Utility::isConflictFile(item->_file)
` into a variable, but, I am not sure how to name it. Something like **shouldAddErrorActivity**?

Reply:
@allexzander Of course you're right. I updated the construct and I was again to fast with squashing. I promise you the next time I will not squash it;)